aep/general/0202/aep.md.j2

```diff
@@ -0,0 +1,52 @@
+# Fields
+
+The [`buf.validate.field`][protovalidate] type, through its accompanying
+extension `buf.validate.field`, enriches a field's schema beyond the basic
+name and type information.
+
+## Guidance
+
+Decorating a field with `buf.validate.field` is only necessary when a field
+consistently follows a set pattern.
+
+#### Format Compatibility
+
+Adding a format specifier to an existing, unspecified field **is not** backwards
+compatible, *unless* the field in question has always conformed to the format
+being specified.
+
+Changing an existing format specifier to a different one in all cases **is not**
+backwards compatible.
+
+## Rationale
+
+#### Why add a format specifier?
+
+The format of a primitive-typed field can be critical to its usability. Some
+programming languages may convey a specific type format as a standalone type,
+as Java does with [UUID][java uuid]. Most have specific structural requirements
+that are validated by the service, so conveying the format to the user ahead of
+time is critical to their experience.
+
+#### Why discourage primitive equality comparisons?
```

Comment:
It seems like there should be actual guidance that this explains.  I think the rationale makes sense, we just need the corresponding guidance.

Reply:
Yeah, this is too vague as-is. The biggest pieces of guidance I can think of are:

* Fields that follow a pattern must have that pattern specified as proto-validate
* Both the request + response fields must have the validation + and the validations must match (if applicable)
* Returning a value that does not match the validation must return an error message.

Any others you can think of?

Reply:
I meant specifically for "Why discourage primitive equality comparisons?"

Which guidance discourages primitive equality comparisons?

Reply:
Yeah, we should get rid of this. 

Overall, the original guidance on this AIP was very weak, but the rationalizations were very well documented. My goal is to have very strong guidance that mostly rationalizes itself. 

We shouldn't need these "why do this" sections if the rules are explanatory enough.

Reply:
I didn't even mean to say that - I'm all for including rationale sections, actually.

But the rationale has to connect to some actual specific guidance, and I can't figure out what guidance this rationale connects to.